### PR TITLE
Add queries and update animals table

### PR DESCRIPTION
**Pull request by Thinus01**
## General requirements
- Correct Gitflow.
- Documented work in a professional way.

## Project requirements
- Added a column called species.
- Inserted data into the database.
- Wrote the following queries and got the following results:

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_k
 INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (2, "Gabumon", "2018-11-15", 2, True, 8kg);
 INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (3, "Pikachu", "2021-01-07", 1, False, 15.04kg);
 INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (4, "Devimon", "2017-05-12", 5, True, 11kg);
+INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (5,"Charmander","2020-02-08",0,False,11,NULL);
```
Comment (BregornOriginal) [inline data.sql]: - Good work creating a new animal, but there's an error while you add it. The amount of Kg is a `-11` negative number, not a positive. Kindly change it 💯

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_k
...
+INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (6,"Plantmon","2021-11-15",2,True,5.7,NULL);
```
Comment (BregornOriginal) [inline data.sql]: - Good work creating a new animal, but there's an error while you add it. The amount of Kg is a `-5.7` negative number, not a positive. Kindly change it 💯

File: data.sql [hunk 3]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_k
...
+INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (7,"Squirtle","1993-04-02",3,False,12.13,NULL);
```
Comment (BregornOriginal) [inline data.sql]: - Good work creating a new animal, but there's an error while you add it. The amount of Kg is a `-12.13` negative number, not a positive. Kindly change it 💯

File: data.sql [hunk 4]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_k
...
+INSERT INTO animals (id, name, date_of_bith, escape_attempts, neutered, weigth_kg) VALUES (8,"Angemon","2005-06-12",1,True,45,NULL);
```
Comment (BregornOriginal) [inline data.sql]: - Good work creating a new animal, but there's an error while you add it. The amount of Kg is a `-45` negative number, not a positive. Kindly change it 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @Thinus01  

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✅ Linters are passing
✅ Good commit history
✅ Linters look professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Thinus01 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [ ] @Thinus01 it would be nice if you updated the query screenshots as the changes requested affect the output of the queries.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**